.github/workflows/build-and-test.yml

```diff
@@ -191,3 +191,12 @@ jobs:
       - name: Deploy docs to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v1
+
+  # This is just a 'ghost' workflow to have all the github CI passed
+  build:
+    name: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: build
+        run: echo Hello, world!
```

Comment:
whoot? :D can't we just re-name the DTE-Coordinator to build and remove the ghost one? ^^

Reply:
I have left comments in [the issue](https://github.com/rx-angular/rx-angular/issues/1454).